### ISSUE TITLE
vak train crashes when normalize_spectrograms = false

### ISSUE THREAD

**Issue by NickleDave** [NickleDave/vak#111]
as @yardencsGitHub pointed out in #101 and #110 
(https://github.com/NickleDave/vak/issues/101#issuecomment-596693323)

with the following error
```
Traceback (most recent call last):
  File "/home/nickledave/anaconda3/envs/vak-env/bin/vak", line 11, in <module>
    load_entry_point('vak', 'console_scripts', 'vak')()
  File "/home/nickledave/Documents/repos/vak/src/vak/__main__.py", line 43, in main
    config_file=args.configfile)
  File "/home/nickledave/Documents/repos/vak/src/vak/cli/cli.py", line 21, in cli
    train(toml_path=config_file)
  File "/home/nickledave/Documents/repos/vak/src/vak/cli/train.py", line 114, in train
    [transforms.Lambda(standardize),
  File "/home/nickledave/anaconda3/envs/vak-env/lib/python3.6/site-packages/torchvision/transforms/transforms.py", line 318, in __init__
    assert callable(lambd), repr(type(lambd).__name__) + " object is not callable"
AssertionError: 'NoneType' object is not callable
```

**Comment by NickleDave** [comment]
@yardencsGitHub the problem is that there's no logic to check whether `standardize` is `None`, so the code tries to add a `None` to the `Compose` of transforms.

The readable way to fix it is to have some statement like `if standardize is not None` ... add it to the list of transforms, and then compose them

To fix it for now, I will add a `util` module to the `transforms` sub-package with a `get_defaults` function that encapsulates this logic (that is missing now because I forgot).

Down the road we want the flexibility to specify transforms. To do so, we can add a function to the `util` module like `from_config` or something like that, and have a `Transforms` section in the config file. I will raise a separate issue about this and add it to a roadmap somewhere. But it will be a post-paper thing, unless we really find we need to try different combos of transforms for some reason.

**Comment by yardencsGitHub** [comment]
I suggested a minimal fix (change the None to an identity transform .. in #110 )
will Compose work with that?

**Comment by NickleDave** [comment]
Thank you for your suggestion, I saw it.
I want to make sure the code is readable, that it's clear *why* we do what we do.

I guess you could put Lambda None, but why include a transform that does nothing?

It just adds one extra meaningless computation, and doesn't fix the real problem, that I forgot to include logic for dealing with the case where there's no normalization.

This is a small fix and then I can move on to addressing #110
